language/diem-framework/modules/0L/AutoPay.move

```diff
@@ -236,20 +243,20 @@ address 0x1 {
             0
           }
         } else {
-          // in remaining cases, payment is simple amaount given, not a percentage
+          // in remaining cases, payment is simple amount given, not a percentage
           payment.amt
         };
 
         // check payees are community wallets, only community wallets are allowed
         // to receive autopay (bypassing account limits)
-        if ( amount != 0 && amount <= account_bal){
-          if (borrow_global<AccountLimitsEnable>(Signer::address_of(vm)).enabled) {
-            if (Wallet::is_comm(payment.payee)) {
-              DiemAccount::vm_make_payment_no_limit<GAS>(
+        if (amount != 0 && amount <= account_bal) {
+          // Todo: if and else have the same code
+          if (borrow_global<AccountLimitsEnable>(Signer::address_of(vm)).enabled &&
+              Wallet::is_comm(payment.payee)
+          ) {
```

Comment:
@jamesmeijers I was talking about the code being the same inside the if/else - this code:            
DiemAccount::vm_make_payment_no_limit<GAS>(*account_addr, payment.payee, amount, x"", x"", vm);